.circleci/config.yml

```diff
@@ -510,6 +510,51 @@ jobs:
           command: |
             docker push flowminder/<<parameters.component>>:$CIRCLE_SHA1
 
+  flowetl_build_and_test:
+    docker:
+      - image: circleci/python:3.7
+    working_directory: /home/circleci/project/
+    steps:
+      - checkout:
+          path: /home/circleci/project/
+      - setup_remote_docker
+      - run:
+          name: Log in to docker cloud
+          command: docker login --username $DOCKER_CLOUD_USER --password $DOCKER_CLOUD_PASSWORD
+      - run:
+          name: Build flowetl image
+          command: |
+            echo "Tagging as $CIRCLE_SHA1"
+            docker build -t flowminder/flowetl:$CIRCLE_SHA1 flowetl
+      - run:
+          name: test running with specified UID/GID for flowetl
+          command: |
+            expecteduser=1003
+            expectedgroup=1002
+            gotuser=`docker run --user ${expecteduser}:${expectedgroup} flowminder/flowetl:${CIRCLE_SHA1} bash -c 'id -u'`
+            gotgroup=`docker run --user ${expecteduser}:${expectedgroup} flowminder/flowetl:${CIRCLE_SHA1} bash -c 'id -g'`
+            if [ $gotuser -ne $expecteduser -a  $gotgroup -ne $expectedgroup ]; then
+              exit 1
+            else
+              echo "running as correct user and group"
+            fi
+            # temporary test to make sure dags getting picked up by airflow
```

Comment:
Split to a separate step?

Reply:
will make much more sense when done with pytest...just a placeholder for now so we know it's basically working as expected...I don't want to have to run CI all over again for a cosmetic change to something that will completely go v. soon

Reply:
ALthough.......have to run again to fix typo anyway!

Reply:
As you say, this is going to be replaced by a proper test suite, at which point it will get split into build and one or more test steps _anyway_. Help me understand what the objection to splitting it now is?